elm.json

```diff
@@ -3,11 +3,12 @@
     "name": "eriktim/elm-protocol-buffers",
     "summary": "An Elm implementation of the Protocol Buffers (protobuf) specification",
     "license": "BSD-3-Clause",
-    "version": "1.1.1",
+    "version": "1.2.0",
```

Comment:
please set this back to 1.1.1, I'll update it using the Elm cli

Reply:
done